File: src/main/java/cn/chahuyun/economy/entity/rob/RobInfo.java
package cn.chahuyun.economy.entity.rob;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;

import java.util.Date;

@Getter
@Setter
@Entity(name = "RobInfo")
@Table(name = "RobInfo")
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RobInfo {
    /**
     * 用户QQ号
     */
    @Id
    private Long userId;

    /**
     * 今日时间
     */
    private Date nowTime;

    /**
     * 被抢劫次数
     */
    private Integer beRobNumber;

    /**
     * 抢劫成功次数
     */
    private Integer robSuccess;

    /**
     * 打人成功次数
     */
    private Integer hitSuccess;

    public Date getNowTime() {
        return nowTime;
    }
}

File: src/main/kotlin/cn/chahuyun/economy/entity/rob/RobInfo.kt
package cn.chahuyun.economy.entity.rob

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import java.util.*

@Entity(name = "RobInfo")
@Table(name = "RobInfo")
class RobInfo(
    /**
     * 用户QQ号
     */
    @Id
    var userId: Long? = null,

    /**
     * 今日时间
     */
    var nowTime: Date? = null,

    /**
     * 被抢劫次数
     */
    var beRobNumber: Int? = null,

    /**
     * 抢劫成功次数
     */
    var robSuccess: Int? = null,

    /**
     * 打人成功次数
     */
    var hitSuccess: Int? = null
)
